README.md

```diff
@@ -91,16 +105,16 @@ In order for all the tests to pass, you will need to create a new (empty) projec
 $ make testacc
 ```
 
-Sample Terraform configuration
-------------------------------
+Sample Terraform configuration when testing locally
+------------------------------------------------------------
 Below is an example configuration to initialize provider and create a Virtual Machine instance
 
 ```sh
 $ cat provider.tf
 terraform {
   required_providers {
     cloudstack = {
-      source = "cloudstack/cloudstack"
+      source = "localdomain/provider/cloudstack"
```

Comment:
The previous one worked for when I was testing locally @kiranchavala , may you want to add what localdomain/provider means ?

Reply:
@harikrishna-patnala 

after building the terraform provider locally using "make build",  the generated  go binary  will contain the new code changes 

To  test  the changes we have to use 
source = "localdomain/provider/cloudstack


If we use " source "cloudstack/cloudstack" , 

when "terraform init" is called terraform will download the binary from terraform registry https://registry.terraform.io/providers/cloudstack/cloudstack/latest  

and place it /.terraform.d/plugins/registry.terraform.io/cloudstack/cloudstack/0.4.0/linux_amd64/terraform-provider-cloudstack_v0.4.0/

If replace the binary with the binary we had built  in the same path and do a "terraform apply" , you will encounter a checksum error as terraform.lock.hcl will be different 

we will hit 

if we try to replace

Reply:
thanks @kiranchavala I used to delete the lock file to test my changes, so that way it worked.